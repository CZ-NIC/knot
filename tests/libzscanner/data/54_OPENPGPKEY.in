$ORIGIN	.
$TTL	1

; OPENPGPKEY is the same as DHCID, so there are differences and basics only.

; OK
@	OPENPGPKEY	AA==		; The simplest variant
@	TYPE61		\# 1 00		; TYPE + Hexadecimal rdata
@	TYPE61		AA==		; TYPE
@	openpgpkey	AA==		; Type in lower-case

; KO
@	OPENPGPKEY
