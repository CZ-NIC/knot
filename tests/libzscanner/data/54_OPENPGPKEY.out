OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003D
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003D
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003D
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003D
RDATA=00
------
WARNG=ZS_BAD_RDATA
------
